js/views/modals/about/About.js

```diff
@@ -30,7 +31,7 @@ export default class extends BaseModal {
       Help,
     };
 
-    this.currentTabName = 'Story';
+    this.currentTabName = this.tabViews[opts.initialTab] ? opts.initialTab : 'Story';
```

Comment:
If we're already defaulting the initalTab up in the opts, can't this just be?

`this.currentTabName = opts.initialTab`

Reply:
I think I see why you did that, but I think the protection would need to be in the `selectTab` function since selecting a tab may not always go through the constructor.

```
  selectTab(tabViewName) {
    let tabView = this.tabViewCache[tabViewName];

    if (!tabView && !this.tabViews[tabViewName]) {
      throw new Error('You are attempting to select an invalid tab.');
    }
    ...
```

Reply:
Yeah, good point.